_component_design/tooltip.md

```diff
@@ -13,28 +13,292 @@ last-modified: 2017-08-17
 
 {% include toc.html %}
 
-Use the tooltip when you want to convey brief snippets of information, to
-explain an element a bit more, or to display truncated text.
+## Introduction
+{: .hxSectionTitle}
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+{: .hxSubSectionTitle}
+
+Use a tool tip when you need to provide in-context help information. For example, when describing what will happen when a workflow is initiated. A tool tip can also be used to reveal hidden information, provide extra or supplemental information to users, when features could be misinterpreted, or an item on a page does not have a text label.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Meet the tooltip pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-hero.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+### When not to use
+{: .hxSubSectionTitle}
+Do not use a tooltip if users must repeatedly refer to a tooltip to understand an option or to complete a workflow. We recommend you do not use a tooltip when users needs to interact with the tip's content, or when users integrate with the site on a mobile device.
+
+### Best practice
+{: .hxSubSectionTitle}
+Tooltips are perfect for short sentences and sentence fragments that assist the user in their understanding. The ideal tooltip is not obvious, nor does it repeat content that is already visible on the site. Place a tooltip in a subtle location, and ensure it does not interfere with the user or obscure any objects of interest. Use the tooltip on icon  images, text, and table rows. The maximum width for the tooltip is 400px, ensure there are no interactive controls inside the tooltip.
```

Comment:
Bullet these best practices, since they are distinct chunks of different concepts.

Reply:
updated. Alex and I thought the bullet points were getting a bit overdone so we converted most of the doc into paragraph form. I like the results but agree the best practice section lends itself to `li` format.